### PR TITLE
Keep returning only PAM_IGNORE from pam_motd

### PR DESCRIPTION
**Pull request by rbalint**
Follow-up for c81280b16e1831ab0bdd0383486c7e2d1eaf1b5e.

### CODE REVIEW

**Comment by ldv-alt** [comment]
On Sat, Dec 14, 2019 at 02:22:21AM -0800, Balint Reczey wrote:
> Follow-up for c81280b16e1831ab0bdd0383486c7e2d1eaf1b5e.
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/linux-pam/linux-pam/pull/157

I suppose errors returned by pam_putenv shouldn't be ignored.
What do you think about the following fix:

-    return retval;
+    return retval == PAM_SUCCESS ? PAM_IGNORE : retval;

**Comment by rbalint** [comment]
OK, I've also updated the man page.

File: modules/pam_motd/pam_motd.8.xml [hunk 1]
```diff
@@ -132,11 +132,27 @@
   <refsect1 id='pam_motd-return_values'>
     <title>RETURN VALUES</title>
     <variablelist>
+      <varlistentry>
+        <term>PAM_ABORT</term>
+        <listitem>
+           <para>
+             Not all relevant data or options could be gotten.
```
Comment (t8m) [inline modules/pam_motd/pam_motd.8.xml]: got or obtained - although sometimes used gotten is weird.

Comment (rbalint) [reply modules/pam_motd/pam_motd.8.xml]: I agree. I copied this over from `pam_env` and the same phrasing is present in other modules, too, thus I did not want to diverge.

**Pull request merged**